proto/v1alpha1/rode_policy.proto

```diff
@@ -171,3 +171,23 @@ message PolicyEntity {
   // Created represents when this policy version was stored. Policy contents are immutable, so there is no corresponding Updated field.
   google.protobuf.Timestamp created = 5;
 }
+
+// PolicyEvaluation describes the result of a request to evaluate a particular resource version against a specific policy.
+// This is a child of ResourceEvaluation.
+message PolicyEvaluation {
+  // Id represents the unique identifier (UUID) for this particular policy evaluation.
+  string id = 1;
+
+  // ResourceEvaluationId represents the unique identifier (UUID) of the resource evaluation that triggered this policy evaluation.
+  string resource_evaluation_id = 2;
+
+  // Pass represents the overall status for this policy evaluation. This is determined by looking at the result of each rule
+  // within the policy and performing an AND on each one.
+  bool pass = 3;
```

Comment:
In `EvaluatePolicy`, at least for the moment, this is populated by a rule called [`pass` in each policy](https://github.com/rode/demo/blob/dceffb2da5c219dd506b7d8ab46777373fa7cd5f/tf/modules/rode/policies/harbor.rego#L3-L5)

https://github.com/rode/rode/blob/17992895bc00d4249f200705f20240d0db8cb4ef/pkg/policy/manager.go#L629

Reply:
Yeah, so `pass` would be duplicated between this and the `EvaluatePolicyResult`.  I probably only need it in one spot.  I think I'll revisit this and make my own proto for the result here, or move some of the fields within `EvaluatePolicyResult` to the top level of `PolicyEvaluation`